tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/XUnitEvpTests.cs

```diff
@@ -90,6 +101,11 @@ public void SubmitTraces(string packageVersion)
                         // Check Module
                         Assert.True(tests.All(t => t.TestModuleId == testSuites[0].TestModuleId));
 
+                        // Check Session
+                        Assert.True(tests.All(t => t.TestSessionId == testSuites[0].TestSessionId));
+                        Assert.True(testSuites[0].TestSessionId == testModules[0].TestSessionId);
+                        Assert.True(testModules[0].TestSessionId == sessionId);
```

Comment:
You've tricked me into writing these for you, haven't you

```suggestion
                        tests.Should().OnlyContain(t => t.TestSessionId == testSuites[0].TestSessionId));
                        testSuites[0].TestSessionId.Should().Be(testModules[0].TestSessionId));
                        testModules[0].TestSessionId.Should().Be(sessionId);
```

Reply:
done in [44a7b7d](https://github.com/DataDog/dd-trace-dotnet/pull/3389/commits/44a7b7d87f83fa43b88233fb52acbb295aad2d15)